packages/api-cardano-db-hasura/test/util.ts

```diff
@@ -8,7 +8,7 @@ export const testClient = {
     'http://localhost:3100'
   ),
   testnet: buildClient.bind(this,
```

Comment:
```suggestion
  preprod: buildClient.bind(this,
```

Reply:
Updated in [7142e2b](https://github.com/input-output-hk/cardano-graphql/pull/800/commits/7142e2b752453f0f6a6ef7677dd1d4e1f6389ee9)